ldk/javascript/examples/self-test-loop/src/tests/screen/index.ts

```diff
@@ -1,6 +1,8 @@
 /* eslint-disable no-async-promise-executor */
-import { screen, whisper, window } from '@oliveai/ldk';
+import { network, screen, whisper, window } from '@oliveai/ldk';
 import { OcrEvent } from '@oliveai/ldk/dist/screen/types';
+import * as base64 from 'byte-base64';
```

Comment:
just fyi you can also do 

`import Buffer from 'Buffer'`

instead of using byte-base64 you could

Reply:
@racary-olive  Yeah, we can use either one! :) Actually Jeff and I paired up a little bit yesterday on this and we used this. I don't have a strong opinion on using `Buffer.from ` or `base64` tho. Do you feel like we want to use `Buffer`? :)

Reply:
I was just thinking since its a new package do we need to get it approved for use/theres a native node package that does the same thing

Reply:
I agree with Ryan. I think we need to add `Buffer` to package too. Maggie and I have tested to use `Buffer` straight in the code, but received  `Buffer is not defined` error, adding `Buffer` to package will fix this issue.

Reply:
Just updated @racary-olive